res/values-zh-rCN/strings.xml

```diff
@@ -233,7 +233,7 @@
     <item quantity="other">%1$d秒后重试</item>
   </plurals>
   <plurals name="lockscreen_access_pattern_failure_left_try_times">
-    <item quantity="other">密码错误，还可以再输入<xliff:g id="times">%1$d</xliff:g>次</item>
+    <item quantity="other">密码错误，还可以再输入%1$d次</item>
```

Comment:
Why has this patch introduced so many i18n modifications, please rebase the develop branch.